addons/binding/org.openhab.binding.tesla/OSGI-INF/TeslaCommands.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.tesla.command.TeslaCommandExtension">
+
+   <scr:implementation class="org.openhab.binding.tesla.command.TeslaCommandExtension"/>
+
+   <scr:reference bind="setStorageService"
```

Comment:
We usually also always define an unbind method `unsetStorageService`, in which we unset the internal field. Could you add that as well?